### PR TITLE
Add a texture pack selector 

### PR DESCRIPTION
**Pull request by metalcupcake5**
Adds the texture pack selector.
I would like everyone to review this, if possible.
Screenshots:
On home screen
image
On profile screen
image

### CODE REVIEW

**Comment by WarpWing** [comment]
I should be asleep lmao. Wait, what about Ender and other community textures?

**Comment by boblovespi** [comment]
we don't have a pack from ender yet

**Comment by WarpWing** [comment]
Ah, how easy would it be to deploy a new pack into the ResPack Switcher

**Review by WarpWing — APPROVED** [review]
Don’t know much JS but CSS looks good to me. I approve

**Comment by WarpWing** [comment]
Good night, I’m going to sleep for now

**Review by boblovespi — CHANGES_REQUESTED** [review]
image
image
breaks on devices with small/narrow screens

File: includes/footer.ejs [hunk 1]
```diff
@@ -17,6 +17,25 @@
...
+        if(infoBox.classList.contains('packs-opened'))
+            infoBox.classList.remove('packs-opened')
+        else
+            infoBox.classList.add('packs-opened');
+    }
+
+    function switchPack(pack = false){
+        console.log(pack)
+        let baseURL = window.location.protocol + "//" + window.location.host + "/" + window.location.pathname.split('/')[1] + "/" + window.location.pathname.split('/')[2] + "/" + window.location.pathname.split('/')[3]
+        if(pack != false){
```
Comment (boblovespi) [inline includes/footer.ejs:32]: why the unnecessary `!= false`

**Comment by WarpWing** [comment]
If I wanted to add a new texture pack, how would I deploy it into the texturepack switcher?

**Comment by metalcupcake5** [comment]
@WarpWing add it like a normal texture pack

**Pull request merged**